connect/src/main/java/com/stripe/android/connect/webview/StripeDownloadManager.kt

```diff
@@ -0,0 +1,62 @@
+package com.stripe.android.connect.webview
+
+import android.app.DownloadManager
+import android.content.Context
+import android.database.Cursor
+import android.net.Uri
+import android.webkit.URLUtil
+
+/**
+ * Provides an interface for various download and file operations. Useful for mocking in tests.
+ */
+internal interface StripeDownloadManager {
+    /**
+     * Returns the ID of the download, or null if the download could not be started.
+     */
+    fun enqueue(request: DownloadManager.Request): Long?
+
+    /**
+     * Returns a [DownloadManager.Request] for the given URI.
+     */
+    fun getDownloadRequest(uri: String): DownloadManager.Request
```

Comment:
does this need to be part of the interface? sort of feels like an implementation detail